hangman.rb

```diff
@@ -0,0 +1,184 @@
+require "Colorize"
+
+class Hangman
+
+  def initialize
+
+    #this runs at the beginning and calls lots of other methods
+
+    # initialized variables
+    @answer = pick_answer
+    # User has 7 wrong guesses before game over
+    @no_of_guesses = 0
+    @blanks_array = answer_to_blanks
+    @guessed_letters = [ ]
+
+    # methods we need to call in the flow of the game
+    start
+    art
+    get_guess
+
+  end
+
+  def start
+    puts "Welcome to Hangman! Type " + "'quit'".colorize(:red) + " at any time to exit."
+  end
+
+
+# creates an answer from the array that is randomized
+  def pick_answer
+    # randomly pick the answer from this array
+    library = ["cat", "dog", "mouse", "letter", "elephant",
```

Comment:
I agree!  I could take the library array and plop it into a constant at the beginning.